thirdeye-server/src/test/resources/alertEvaluation.json

```diff
@@ -99,7 +99,5 @@
         }
       ]
     }
-  },
-  "start": "${start}",
-  "end": "${end}"
+  }
```

Comment:
glitch? how does alert evaluation work without start and end?

Reply:
The `DetectionPipelineContext` passed to planExecutor$runPipeline contains a detection Interval.
It is consumed by the DataFetcher via the __timeFilterMacro but could be by other ways.

I did this change a long time ago now, I think the main pros at the time were:
- detectionInterval is required by almost all operators, and we must ensure it's the same.
- detectionInterval should be compatible with:
-- timezone
-- granularity bucket rouding
-- completeness delay
-- re-play (recent, for data mutability)